tests/security/cc/ip_eb_tables.pm

```diff
@@ -24,6 +24,9 @@ sub run {
 
     select_console 'root-console';
 
+    # install command ping
+    zypper_call('in iputils');
+
     # Configure bridge for ip_eb_tables workaround
```

Comment:
Xiaojing,
Bridge setup is already there for x86-64 and aarch64, can you please enhance this script as well?
Any, you can file a new PR for it.

Reply:
Did in another commit